pipeline.go

```diff
@@ -82,8 +83,18 @@ func (p *Pipeline) gather(
 
 	p.publishEvent(eventChan, RepositoryListPipelineEvent{repositories})
 
-	for _, repository := range repositories {
-		output <- repository
+	ignored := 0
+
+	for index, repository := range repositories {
+		if limit > 0 && index >= limit {
+			ignored++
+		} else {
+			output <- repository
+		}
+	}
+
+	if ignored > 0 {
+		log.Warnf("Limit reached of repositories! %d repositories ignored", ignored)
```

Comment:
I would say : "Limit reached for number of repositories", and perhaps say both : "collected X repos, ignored Y repos." WDYT ?

Reply:
The output is now:
```
WARN[0000] Limit reached for number of repositories     
WARN[0000] Collected 2 repos, ignored 7 repos.   
```

Reply:
LGTM. Did you see the lint error, I am not sure to see how you could do differently ? If you think we can ignore, please merge ;)  
I'll do a general update of the README in a separate PR :)